packages/core/src/scss/components/checkout/checkoutRemote/_checkoutRemote.scss

```diff
@@ -15,22 +15,23 @@
     > .AmazonPayContainer {
         width: remCalc(200px);
     }
+}
 
-    > #applepayCheckoutButton button,
-    > #braintreepaypalcreditCheckoutButton,
-    > #braintreepaypalCheckoutButton {
-        width: 160px;
-    }
+#applepayCheckoutButton button,
+#braintreepaypalcreditCheckoutButton,
+#braintreepaypalCheckoutButton {
+    width: 160px;
+}
 
-    > #applepayCheckoutButton button {
-        background-color: #000;
-        background-image: -webkit-named-image(apple-pay-logo-white);
-        background-position: 50% 50%;
-        background-repeat: no-repeat;
-        background-size: 100% 60%;
-        border-radius: $global-radius;
-        height: $wallet-button-height;
-    }
+#applepayCheckoutButton button {
+    background-color: #000;
+    background-image: -webkit-named-image(apple-pay-logo-white);
+    background-position: 50% 50%;
+    background-repeat: no-repeat;
+    background-size: 100% 60%;
+    border-radius: $global-radius;
+    height: $wallet-button-height;
+    padding: spacing('single');
```

Comment:
Removed unnecessary nesting and adjusted Apple Pay button size in the customer step.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/88361607/221100591-792e0a11-f2c4-4414-b205-88bc3668e03c.png">